Command/IndexImportCommand.php

```diff
@@ -53,13 +59,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $manager = $this->getManager($input->getOption('manager'));
 
+        $options = [];
+        if ($input->getOption('gzip')) {
+            $options['gzip'] = true;
```

Comment:
Yup, regardless of the option's value, I only check the existence of the key 'gzip'. Your idea is nice. I will fix it. Thanks.